server/data/words/german.json

```diff
@@ -404,9 +404,13 @@
     "translation": "always"
   },
   {
-    "word": "tolle",
+    "word": "toll",
     "translation": "amazing"
   },
+  {
+    "word": "Tolle",
+    "translation": "quiff"
```

Comment:
Could you please check the new language contribution guidelines for the 3000 words to be translated and their order?

Reply:
Got it, quiff is not on the list so it shoudn't be in there, am I getting your hint correct?